src/walks.jl

```diff
@@ -1,3 +1,7 @@
+_map(f, x...) = map(f, x...)
+_map(f, x::Dict) = Dict(k => f(v) for (k, v) in x)
```

Comment:
What happens if we get `_map(f, x::Dict...)`? Currently it would result in a MethodError from inside `map`, so ideally we'd at least make a more user friendly message.

Reply:
I extended the method to support multiple dictionaries